### PR TITLE
Adds docs generation infra for VertexAI

### PR DESCRIPTION
**Pull request by morganchen12**

### CODE REVIEW

File: FirebaseVertexAI.podspec
Comment (andrewheard) [inline FirebaseVertexAI.podspec:1]: Should we rename this to something like `FirebaseVertexAI-Docs.podspec` (or `FirebaseVertexAI-Preview-Docs.podspec`) to make it abundantly clear that the SDK is not available via CocoaPods? Potentially adding `Docs` and/or `Preview` to the `s.name` as well.

**Review by andrewheard — CHANGES_REQUESTED** [review]
Sorry for the hassle, would you mind branching off of [`release-10.26`](https://github.com/firebase/firebase-ios-sdk/tree/release-10.26) instead of [`vertexai-preview-0.1.0`](https://github.com/firebase/firebase-ios-sdk/tree/vertexai-preview-0.1.0)? We'll want these changes in the public preview release.

File: FirebaseVertexAI.podspec
Comment (morganchen12) [reply FirebaseVertexAI.podspec:1]: I think changing the `s.name` will cause the generated docs dir to have the name in there as well so I won't touch that. But I'll rename the file. As long as we don't push to CocoaPods specs no one should be able to use it anyway.

**Comment by morganchen12** [comment]
I'll make a new PR since this one is branched off the preview branch.

**Pull request closed without merging**